sktree/tree/tests/test_tree.py

```diff
@@ -44,3 +60,21 @@ def test_unsupervisedtree():
 
     # a single decision tree does not fit well, but should still have a positive score
     assert score > 0.05
+
+@pytest.mark.parametrize("name,Tree", CLF_TREES.items())
+def test_iris(name, Tree):
+    # Check consistency on dataset iris.
+    for criterion in CLF_CRITERIONS:
+        clf = Tree(criterion=criterion, random_state=0)
+        clf.fit(iris.data, iris.target)
+        score = accuracy_score(clf.predict(iris.data), iris.target)
+        assert score > 0.3, "Failed with {0}, criterion = {1} and score = {2}".format(
+            name, criterion, score
+        )
+
+        clf = Tree(criterion=criterion, max_features=2, random_state=0)
```

Comment:
What is the motivation for setting `max_features=2`?

Reply:
I dropped this arg